src/routers/posts.router.js

```diff
@@ -1,6 +1,84 @@
 const express = require('express');
+const Post = require('../entities/post.entity');
+const { createPost, readPosts, updatePost, deletePost } = require('../repository/posts.repository');
+const PostDTO = require('../dtos/postDTO.dto');
+const { route } = require('./index.router');
 const router = express.Router();
 
 //CREATE YOUR REST API UNDER HERE
+// Create A Post
+router.post('/post', async (req, res) => {
+    const postDTO = req.body;
```

Comment:
please use `hasMissingKey` from 808067b62139b237aee91523800674bcd05bcaaf
to do validation on postDTO object.

you can cherry-pick 808067b62139b237aee91523800674bcd05bcaaf . Then see the example in 2b4fb8a1f234db7ed9f638d190f5da15b92e7ec7 to know how to use it

**Applicable to all endpoints